src/main/js/store/actions/distributionConfigs.js

```diff
@@ -173,7 +195,12 @@ export function saveDistributionJob(config) {
                     if (response.ok) {
                         dispatch(saveJobSuccess(responseData.message));
                     } else {
-                        handleFailureResponse(DISTRIBUTION_JOB_SAVE_ERROR, dispatch, responseData, response.status);
+                        const defaultHandler = () => jobError({
+                            type: DISTRIBUTION_JOB_SAVE_ERROR,
+                            ...responseData
+                        });
+                        const handler = createErrorHandler(DISTRIBUTION_JOB_SAVE_ERROR, defaultHandler);
+                        dispatch(handler(response.status));
```

Comment:
is handler a function here? We're passing a status to handler but it doesn't look like handler expects anything passed to it.

Reply:
createErrroHandler creates a function reference like a lambda expression.  We are assigning handler to the function that is returned by createErrorHandler.